extension/src/ts/contentScript.ts

```diff
@@ -1,13 +1,16 @@
-const getArticleTitle = () : String => {
-    const tag = document.querySelector('div.l-container h1.pg-headline') as HTMLHeadingElement;
-    return tag?.innerText ?? "";
+import { ContentToBackgroundMsg, MsgType } from "./types";
+
+const sendContent = (textToSend: string) => {
+    const msg: ContentToBackgroundMsg = {
+        type: MsgType.PageContent,
+        text: textToSend
+    };
+    chrome.runtime.sendMessage(msg);
 };
 
-(() => {
-    chrome.runtime.onMessage.addListener((msg, sender, callback) => {
-        console.log('Message received from sender: ', sender.id, msg);
-        const articleTitle : String = getArticleTitle();
-        const displayText : String = articleTitle ? `The article title is ${articleTitle}` : "Not a CNN article page";
-        callback(displayText);
-    });
-})();
+if (window.location.href.includes('cnn.com') && window.location.href.includes('index.html')) {
```

Comment:
Might be useful to put a TODO here to note that we plan to add support for additional (or any) sites with future improvements :)